test.py

```diff
@@ -3,9 +3,9 @@
 from wordsoup import wordsoup, txt_read
 
 
-class MainTestCase(unittest.TestCase):
-    def io_test(self):
-        self.assertEqual('Our team had a debate for best names for looping variables ? i won',
+class GroupExerciseTest(unittest.TestCase):
+    def test_io(self):
```

Comment:
glad you got it working alright. Good to know that the python unit test cases require function names that start with test_